tensorflow/python/keras/losses.py

```diff
@@ -433,6 +435,41 @@ def __init__(self,
     super(MeanAbsolutePercentageError, self).__init__(
         mean_absolute_percentage_error, name=name, reduction=reduction)
 
+def call_huber(threshold=1.0):
+  """Returns the huber_fn for the HuberLoss update_state() function"""
+  def huber_fn(y_pred, y_true):
+    """Returns the huber loss for given output and truth label"""
+    step_error = y_true - y_pred
+    is_lesser = step_error < threshold
+    quadratic = square(error) / 2
+    linear = threshold * error - square(threshold) / 2
+    return array_ops.where(is_lesser, quadratic, linear)
+  return huber_fn
+
+
+@keras_export('keras.losses.HuberLoss')
+class HuberLoss(Loss):
```

Comment:
Name of the function should not be `HuberLoss` since it is already acquired by the huber loss function.

Reply:
Okay. I will change it and add some more checks to the code before adding it to addons.